src/components/pages/photos/elements/RoverImage.js

```diff
@@ -1,14 +1,41 @@
 import React, { Component } from 'react';
 import CameraNavbar from './cameraNavbar';
-import nisa from '../../../../assets/images/nisa.JPG';
 import '../PhotoPage.css';
+import RoverImageButton from './RoverImageButton';
 
 export default class RoverImage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: 0,
+    };
+  }
+
+  incrementIndex() {
+    this.setState(state => (
+      {
+        index: Math.min(state.index + 1, this.props.photoUrlArray.length - 1),
+      }
+    ));
+  }
+
+  decrementIndex() {
+    this.setState(state => (
+      {
+        index: Math.max(0, state.index - 1),
+      }
+    ));
+  }
+
+
   render() {
     return (
       <div className="rover-image">
-        <img className="camera-capture" src={nisa} alt="Mars" />
+        <RoverImageButton className="leftButton" direction={'<'} onClick={() => this.decrementIndex()} />
+        <img className="camera-capture" src={this.props.photoUrlArray[this.state.index]} alt="Mars" />
+        <RoverImageButton className="rightButton" direction={'>'} onClick={() => this.incrementIndex()} />
         <CameraNavbar />

```

Comment:
I think the `CameraNavbar` sticks out a bit as not being directly related to displaying the photo.

Perhaps we should pull it up into the `Rover` element, so the `Rover` would be: 

```
<div className="rover-container">
    <RoverImage photoUrlArray={dummyPhotoUrlArray} />
    <CameraNavbar />
    <AboutRover />
</div>
```